fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabaseRunner.java

```diff
@@ -50,7 +50,8 @@
 /**
  * The standard implementation of {@link FDBDatabaseRunnerInterface}.
  */
-@API(API.Status.MAINTAINED)
+// TODO: Rename this class to FDBDatabaseRunnerImpl in next minor release.
```

Comment:
It might be good to put this in the Javadoc so that callers of this class could see it.